wiki/History_of_osu!/2010/en.md

```diff
@@ -1 +1,68 @@
 # History of osu! 2010
+
+## January
+
+On the [Wai-con](https://en.wikipedia.org/wiki/Wai-Con) anime convention in Perth, osu! had an arcade booth which featured a fully self-contained arcade machine running a touch-based version of osu! on it called osu!arcade. Although playing too much on it could lead to sore fingers as some people had criticised, the prototype containing custom-made hardware and a built-in touchscreen was well-received by the community.[^wai-con]
```

Comment:
this reads like it may benefit from simplification
```suggestion
The [Wai-con](https://en.wikipedia.org/wiki/Wai-Con) anime convention in Perth had osu!arcade, a fully self-contained arcade machine running a touch-based version of osu!. Although playing too much on it could lead to sore fingers as some people had criticised, the prototype containing custom-made hardware and a built-in touchscreen was well-received by the community.[^wai-con]
```